lectures/getting_started_julia/getting_started.md

```diff
@@ -302,12 +302,9 @@ We will explore these sorts of features, and how to use them for your own projec
 
 
 (julia_environment)=
-# Interacting with Julia
+## Interacting with Julia
 
-
-## Overview
```

Comment:
@jbrightuniverse we may want to dedent the remaining titles within that section.

Reply:
I presume the following `overview` is nested withn `Interacting with Julia`

Reply:
and by dedent I mean add `#` to it's title level :-)

Reply:
Do you mean add an extra `#` to each title? Or remove one `#`?

Reply:
ah whoops didn't refresh, got it

Reply:
I mean add one to make it a lower level (i.e. `##` becomes `###`) if the sub-sections below belong to that section

Reply:
@mmcky in the current version I eliminated the `Overview` title entirely since `Overview` is only used at the top of lectures - I had it merged with the `Interacting with Julia` title. So perhaps we don't need to change this?

Reply:
![image](https://user-images.githubusercontent.com/30967260/132292839-443ac1b9-f3cc-4c1f-a625-58343c56090d.png)

This is what it looks like right now.

Reply:
oh I follow -- thanks @jbrightuniverse. So there aren't really any nested sub-sections. Thanks.